pylint/checkers/refactoring.py

```diff
@@ -102,6 +103,10 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                   'weird bugs in your code. You should always use parentheses '
                   'explicitly for creating a tuple.',
                   {'minversion': (3, 0)}),
+        'R1708': ("The mutable sequence (%s) is modified within the loop",
+                  'mutable-sequence-modified-in-loop',
+                  'Modification within the cycle may cause each iteration to'
```

Comment:
The description is a bit confusing to me. Let's rephrase it to ``Modifying a mutable object while iterating through it can result in unexpected behaviour. This message is emitted whenever a case like this is detected``.